articles/openshift/support-policies-v4.md

```diff
@@ -10,26 +10,63 @@ ms.date: 03/05/2021
-* Don't remove or modify Azure Red Hat OpenShift service logging (mdsd pods).
-* Don't remove or modify the 'arosvc.azurecr.io' cluster pull secret.
+* Don't remove or modify any resources in any namespace beginning with `openshift-azure`
+* Don't run workloads in the kube-system namespace or any namespace beginning with "openshift-".
+* Don't remove or modify the 'arosvc.azurecr.io' cluster pull secret.  This secret can be seen with `oc -n openshift-config get secret pull-secret`.
+* Customer workloads must not be scheduled on the master nodes to prevent any resource contention with the OpenShift master components.
+* Don't set any unsupportedConfigOverrides options. Setting these options prevents minor version upgrades.
+* A pod disruption budget should not set a `maxUnavailable` of 0% or 0, or a `minAvailable` of 100% or equal to the number of replicas.  This can block nodes from being drained and upgraded.
+* Don't create or enable any Feature Gates
+* Don't create or enable the cluster-wide proxy
+* Don't modify the audit log policy
+* Don't modify or remove the OpenShift SDN network or attempt to use another SDN provider.
+* Don't modify the cluster network OpenShift SDN configuration.
+* Azure Red Hat OpenShift does not support adding an additional network such as a bridge, host-device, ipvlan, or macvlan network.
+* The cluster version resource should only use stable release channels for upgrades.  There is no support for non-stable channels.  The channel can be seen by running `oc get clusterversion version -o jsonpath="{.spec.channel}"`.  In new cluster installations, the channel may not be set yet.
+* Any workloads that are run on the cluster including operators from OperatorHub must not affect running OpenShift components and cluster operators from being available.
+
+### Azure Resources
+
+* Don't remove the network security groups from their assigned subnets.
+* Upon installation, all cluster resources are created in a resource group called the cluster resource group.  Don't modify or remove any resources within the cluster resource group.
 * All cluster virtual machines must have direct outbound internet access, at least to the Azure Resource Manager (ARM) and service logging (Geneva) endpoints.  No form of HTTPS proxying is supported.
 * Don't modify the DNS configuration of the cluster's virtual network. The default Azure DNS resolver must be used.
-* Don't override any of the cluster's MachineConfig objects (for example, the kubelet configuration) in any way.
-* Don't set any unsupportedConfigOverrides options. Setting these options prevents minor version upgrades.
 * The Azure Red Hat OpenShift service accesses your cluster via Private Link Service.  Don't remove or modify service access.
 * Non-RHCOS compute nodes aren't supported. For example, you can't use a RHEL compute node.
+* Don't place policies within your subscription that prevents SREs from performing normal maintenance against the ARO cluster.
```

Comment:
```suggestion
* Don't place policies within your subscription or management group that prevent SREs from performing normal maintenance against the ARO cluster.
```

Reply:
Would we need to also clarify that tagging of individual resources is not supported?